### ISSUE TITLE
训练模型需要多长时间？

### ISSUE THREAD

**Issue by kFoodie** [yuankeyi/2019-SOHU-Contest#1]
如题。
我在2080Ti上训练的，不知道大概多久要训练完……
参数如下：
> params = {
            'task': 'train',
            'boosting_type': 'gbdt',
            'objective': 'binary',
            'metric': {'auc', 'binary_logloss'},
            'learning_rate': 0.001,
            'num_leaves': 64,
            'min_data_in_leaf': 170,
            'bagging_fraction': 0.85,
            'bagging_freq': 1,
            'seed':42,
            'device': 'gpu',
            'gpu_platform_id': 0,
            'gpu_device_id': 2,
            'max_bin': 63,
            'gpu_use_dp': 'false',
            'num_thread': 24,
            }

**Comment by yuankeyi** [comment]
我只能说下我之前的情况。
开了50个核的CPU跑了8小时

发自袁可一

**Comment by kFoodie** [comment]
我这快训练完了。想问一下，有什么强特征可以用来做候选词的排序或者选择吗？

**Comment by yuankeyi** [comment]
目前我也没怎么发现，你可以帮我看一下hhh

发自袁可一

**Comment by kFoodie** [comment]
哈哈，拿你的模型来融合，希望有好的效果。这会决赛的时候，lgb全挂了= =。。

**Comment by yuankeyi** [comment]
如果有效果或者没效果，请和我都说一声，谢谢！

发自袁可一

**Comment by kFoodie** [comment]
没啥效果。实体只有0.58……弃坑了……
